Editor/ExportUtility.cs

```diff
@@ -667,6 +678,14 @@ static void BakeNonstandardRenderers(DynamicObject rootDynamic, List<BakeableMes
                 }
             }
 
+#if C3D_TMPRO
+            // For "just TextMeshPro" - we create a Canvas to ease export
+            // Delete the Canvas later
+            foreach (var v in TextMeshPros)
+            {
+                ExportQuad(v.gameObject, meshes, ExportQuadType.TMPro);
```

Comment:
The `ExportQuad` function has been updated to be a little more generic to handle both `Canvases` and raw `TMPro` - can expand in the future I suppsoe